src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -216,7 +217,7 @@ export const validateAddress = (
         route,
         method,
         fieldName,
-        payload,
+        { ...suggestedAddresses[0], id: payload?.id },
```

Comment:
Maybe we can place this up in a variable so its more readable, other than that looks good as long as we always expect a value in the `suggestedAddresses` array.